presto-main/src/test/java/io/prestosql/block/TestInt128ArrayBlock.java

```diff
@@ -82,6 +85,19 @@ public void testCompactBlock()
         testIncompactBlock(new Int128ArrayBlock(valueIsNull.length - 2, Optional.of(valueIsNull), longArray));
     }
 
+    @Test
+    public void testIsDistinctFrom()
+    {
+        long[] arr1 = {112L, 0L}; // DECIMAL '1.12'
```

Comment:
These can be inlined. There's no real benefit in having them as standalone variables.

Reply:
I'll take care of it.